src/App.js

```diff
@@ -69,6 +71,18 @@ export default function App() {
                   />
                 )}
               </Route>
+              <Route path="/pet/:id/update-pet">
+                {user.id ? (
+                  <EditPetForm pets={pets} deletePet={deletePet} />
```

Comment:
Some how deletePet doesn't feel right ;) Feels like giving it to the shelter ... Just kidding.

Reply:
I know, right! I didn't really want to build that feature, but I guess we have to be practical :(